src/services/shared/Timetable.ts

```diff
@@ -36,3 +36,9 @@ export enum TimetableClassStatus {
   ONLINE = "En ligne",
   TEST = "ds",
 }
+
+export interface WeekFrequency {
+  textLabel: String,
+  freqLabel: String
+  num: number
+}
```

Comment:
attention, pas de majuscules, c'est autre chose

```suggestion
export interface WeekFrequency {
  textLabel: string,
  freqLabel: string,
  num: number
}
```

Reply:
Ah oui bien vu en effet, résolu avec le dernier commit :+1: